packages/private-build-infra/addon/current-deprecations.ts

```diff
@@ -0,0 +1,14 @@
+export default {
+  DEPRECATE_CATCH_ALL: '99.0',
```

Comment:
What is this one for?

Reply:
It allows you to write something in tests that is only stripped if all deprecations are stripped, we don't use it yet and could remove it but I suspect we will want it for adding tests for the infra.